kobo/worker/logger.py

```diff
@@ -20,12 +20,13 @@
 class LoggingThread(threading.Thread):
     """Send stdout data to hub in a background thread."""
 
-    def __init__(self, hub, task_id, *args, **kwargs):
+    def __init__(self, hub, task_id, buffer_size=256, *args, **kwargs):
```

Comment:
I'm not sure if this is backwards-compatible, due to the following `*args` which are passed into Thread.  Example:

`LoggingThread(some_hub, 123, None, None, 'my-logging-thread')`

In prior kobo version this meant:
- hub=some_hub
- task_id=123
- group (first argument of `Thread` constructor) = None
- target=None
- name='my-logging-thread'

If this change is submitted, won't the meaning of that code change into this? (and will definitely crash)
- hub=some_hub
- task_id=123
- buffer_size=None
- group=None
- target='my-logging-thread'

Put another way: the API was defined as passing the third positional argument onwards into Thread, changing that so it only passes the *fourth* positional argument onwards into Thread is not backwards-compatible. 

Given that the method was defined this way, in order to add a new argument I think you will need to make it a keyword-only argument.

Reply:
Yes, you're correct.  Addressed in 35178ff.